lib/msf/core/exploit/http/client.rb

```diff
@@ -463,10 +463,10 @@ def target_uri
   end
 
   # Returns the complete URI as string including the scheme, port and host
-  def full_uri
```

Comment:
If we are already updating the `full_uri` method maybe it makes sense to also hide port 443 when doing `ssl`

```
def full_uri(custom_uri = nil)
	uri_scheme = ssl ? 'https' : 'http'
	if ssl
		uri_port = rport.to_s == '443' ? '' : ":#{rport}"
	else
		uri_port = rport.to_s == '80' ? '' : ":#{rport}"
	end
	uri = normalize_uri(custom_uri || target_uri.to_s)
	"#{uri_scheme}://#{rhost}#{uri_port}#{uri}"
end
```